nupic/research/frameworks/dynamic_sparse/models/dsnn.py

```diff
@@ -259,18 +259,18 @@ def prune(self, module):
             weight = module.m.weight.clone().detach()
             num_params = module.num_params
             corr = module.get_coactivations()
-            weight_prune_perc = module.weight_prune
+            hebbian_prune_perc = module.hebbian_prune
```

Comment:
It is important to mention these changes in the pull request, the title and text only talks about logging

Reply:
Good point. There's a comment above but I'll add it to the description.

Reply:
Fixed